.github/renovate.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":prHourlyLimit4",
+    ":prConcurrentLimit20",
+    ":pinAllExceptPeerDependencies",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
```

Comment:
Are [semantic commits](https://www.conventionalcommits.org/en/v1.0.0/) something you want to have, @stevespringett?  If they are used, they should be used consistently, otherwise they're kinda pointless. This would also involve stuff like [commit squashing](https://www.conventionalcommits.org/en/v1.0.0/#do-all-my-contributors-need-to-use-the-conventional-commits-specification) when merging PRs etc.

If we don't commit (lol) to using it, it shouldn't be included in the renovate config.

Reply:
I second that! As this was just an initial proposal to see if this was something you would be interested I reused some of my other configs I use personally, which included semantic commits :sweat_smile: 

Depending if you want to commit to it (pun intended) or not I would keep or remove this preset.

Reply:
Okay, will do.
However, on the note of keeping it consistent, the frontend repo seems to not have had Dependabot configured :?